spec/operators/boolean.hrx

```diff
@@ -0,0 +1,75 @@
+<===> whitespace/and/before/sass/input.sass
+$a: b 
+and c
+
+<===> whitespace/and/before/sass/output.css
+
+<===> whitespace/and/before/sass/warning
+WARNING: This selector doesn't have any properties and won't be rendered.
+
+  ,
+2 | and c
+  | ^^^^^
+  '
+    input.sass 2:1  root stylesheet
+
+<===>
+================================================================================
+<===> whitespace/and/after/sass/input.sass
+$a: b and 
+  c
+
+<===> whitespace/and/after/sass/output.css
+
+<===>
+================================================================================
+<===> whitespace/not/after/sass/input.sass
+$a: b not 
+  c
+
+<===> whitespace/not/after/sass/output.css
+
+<===>
+================================================================================
+<===> whitespace/not/before/sass/input.sass
+$a: b 
+not c
```

Comment:
Unary operators shouldn't be tested in the same way as binary operators, since they aren't parsed in the same way.